jest.config.js

```diff
@@ -24,6 +24,8 @@ module.exports = {
 		'^\\.~/(.*)$': '<rootDir>/assets/source/$1',
 		'@woocommerce/settings':
 			'<rootDir>/assets/source/tests/dependencies/woocommerce/settings',
+		'@woocommerce/tracks':
+			'<rootDir>/assets/source/tests/dependencies/woocommerce/tracks',
```

Comment:
:nail_care: 
WDYT of chaning it to 
```suggestion
		'@woocommerce/(settings|tracks)':
			'<rootDir>/assets/source/tests/dependencies/woocommerce/$1',
```

So we will not grow with LOC when we add more and more mocks?

Reply:
Nice suggestion @tomalec 

What about https://github.com/woocommerce/pinterest-for-woocommerce/pull/311/commits/213509745a1b913453084ecae6ee1739cb786977

That way we can also add potential out WC packages.